sdk/src/wallet/operations/syncing/addresses/output_ids/mod.rs

```diff
@@ -230,20 +229,21 @@ impl<S: 'static + SecretManage> Wallet<S> {
     /// return spent outputs separated
     pub(crate) async fn get_output_ids_for_addresses(
         &self,
-        addresses_with_unspent_outputs: Vec<AddressWithUnspentOutputs>,
+        addresses: &[AddressWithUnspentOutputIds],
         options: &SyncOptions,
-    ) -> Result<(Vec<AddressWithUnspentOutputs>, Vec<OutputId>), WalletError> {
+    ) -> Result<(Vec<AddressWithUnspentOutputIds>, Vec<SpentOutput>), WalletError> {
         log::debug!("[SYNC] start get_output_ids_for_addresses");
         let address_output_ids_start_time = Instant::now();
 
-        let mut addresses_with_outputs = Vec::new();
+        let mut addresses_with_unspent_outputs = Vec::new();
         // spent outputs or account/nft/foundries that don't get synced anymore, because of other sync options
+        // TODO: can we rename this to `spent_or_ignored_outputs`? (they are ignored because ^)
```

Comment:
Fine with me if you find that easier to understand

Reply:
not necessarily easier to understand, but more concise than `not anymore synced`

Reply:
But renaming OutputData to OutputWithExtendedMetadata :trollface:

Reply:
There are a few options I considered:
* `OutputData` -> could mean anything and nothing
* `OutputWithExtMetadata` -> kinda looks ugly
* `OutputWithExtData` -> kinda looks ugly and this `Data` thing again that can mean ...
* `OutputWithMetadataExt` -> Ext suffix usually used for traits
* `OutputWithExtendedMetadata` -> long, but most accurate

Reply:
Already done.